codewiki/src/be/cluster_modules.py

```
@@ -44,68 +48,128 @@ def cluster_modules(
     leaf_nodes: List[str],
     components: Dict[str, Node],
     config: Config,
-    current_module_tree: dict[str, Any] = {},
-    current_module_name: str = None,
-    current_module_path: List[str] = []
+    current_module_tree: Dict[str, Any] = {},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)